HIPs/incubation/identus.md

```diff
@@ -0,0 +1,172 @@
+
+## Choosing Between Identus and Aries
+
+When selecting a framework for decentralized identity solutions, consider the following factors:
+- Programming Language Preference: Although writing business logic can be done in any language, if your development team favors Scala (JVM) based backends, Identus offers a natural fit. On the other hand, Aries provides implementations in Python and JavaScript, among others.
+- DIDComm Version: If your project's requirements necessitate DIDComm v2, Identus would be the more suitable choice.
+- Community: To determine whether Identus and Aries suit your project, you must evaluate the community composition. Although Identus aims to be VDR agnostic, it has a more substantial presence within the Cardano communities due to its association with IOG, which offers better synergies for use cases that target the Cardano ecosystem.
+
+# Dependent Projects
+none
+
+# Motivation
+Identus addresses the challenges of building decentralized identity solutions. Traditionally, newcomers to the decentralized identity space face steep learning curves. We aim to ease the task by providing comprehensive components, detailed documentation, and adherence to well-known domain standards. Our technology-agnostic approach reduces friction, while our focus on abstracting complexities of SSI technology allows solution builders to concentrate on innovative use cases across Web2 and Web3 landscapes. Identus strives to provide flexibility by supporting multiple credential formats and verifiable credential exchange options.
+
+Most importantly, we've ensured cohesion by conveniently implementing the most sought-after decentralized identity functionalities in one place, making our offering a solution for those looking to streamline the development of decentralized identity systems.
+
+# Status
+
+Incubation
+
+Today, the Cloud Agent is at Hyperledger as the [Open Enterprise Agent - Lab](https://github.com/hyperledger-labs/open-enterprise-agent).
+The goal is to contribute more components of Atala PRISM under the Identus Project. The other components we want to make part of Identus are:
+- [DIDcomm v2 Mediator](https://github.com/input-output-hk/atala-prism-mediator )
+- [DIDcomm v2 Mediator - Test Suite](https://github.com/input-output-hk/didcomm-v2-mediator-test-suite)
+- [Apollo Cryptographic library](https://github.com/input-output-hk/atala-prism-apollo/)
+- Wallet SDKs
+  - [TS Implementation](https://github.com/input-output-hk/atala-prism-wallet-sdk-ts)
+  - [Swift Implementation](https://github.com/input-output-hk/atala-prism-wallet-sdk-swift)
+  - [KMM Implementation](https://github.com/input-output-hk/atala-prism-wallet-sdk-kmm)
+
+Beyond the components presented here, the proposal extends to other features we plan to develop.
+
+# Solution
```

Comment:
Is there an architecture diagram available showing how all of these different components fit together?

Reply:
I added a diagram to the proposal.